cli/turbo.json

```diff
@@ -3,7 +3,34 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
-      "outputs": ["turbo", "turbo.exe"]
+      "outputs": [
+        // TODO: this gets moved over to ../target, can we omit this?
+        "turbo",
+        // TODO: does this get moved over to ../target also? Can we omit this too?
+        "turbo.exe",
+        // TODO: This is 7GB, do we want to ever even cache the debug build?
+        "../target/debug/**",
+        // TODO: Do we need the whole directory? go-turbo and turbo are ~45mb, the whole directory is 800mb
+        "../target/release/**"
+      ],
+
+      "inputs": [
+        "**/*.go",
+        "**/*_test.go",
+
+        // Rust crates that should trigger a rebuild
+        // TODO: this is not the full list (e.g. pidlock is missing here)
+        // what else do we need here? And should turbo be crawling / using cargo
```

Comment:
One thing I've been meaning to do is rename all of the turborepo crates to be prefixed with `turborepo`, as agreed upon in conversations with web-tooling. That'll make it easy to filter

Reply:
That would be nice, but I assume there will still be crates that both teams wil use (hopefully that's the purpose of monorepo'ing both). I'm not sure how we'll account for that without crawling deps (maybe this is what you meant by "Rust support" in the team meeting?)

Reply:
I think that's a bridge we'll cross when we come to it. We can always come up with another prefix like `shared-`